command/job_action.go

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Small modifications made to the Help text, etc., but otherwise this is the same as what previously existed in action.go